components/Calendar.jsx

```diff
@@ -6,48 +6,70 @@ import "react-big-calendar/lib/css/react-big-calendar.css";
 import CustomToolbar from "./CustomToolbar.jsx";
 import CustomEvent from "./CustomEvent.jsx";
 import Modal from "./Modal.jsx";
+import axios from "axios";
 
 const mLocalizer = momentLocalizer(moment);
 
 const CalendarEvents = () => {
+  axios
+    .get(
+      `https://www.googleapis.com/calendar/v3/calendars/${process.env.NEXT_PUBLIC_GOOGLE_CALENDAR_EMAIL}/events?key=${process.env.NEXT_PUBLIC_GOOGLE_CALENDAR_API_KEY}`,
+      {
+        params: {
+          queryParams:
+            "singleEvents=true&orderBy=startTime&timeMin={current_time_str}&maxResults=1",
```

Comment:
- integrate these directly into the url
- also timeMin={current_time_str} is doing nothing
- remove the max results